google/cloud/connectors/v1/.repo-metadata.json

```diff
@@ -0,0 +1,14 @@
+{
+    "api_id": "connectors.googleapis.com",
+    "api_shortname": "connectors",
+    "client_documentation": "https://cloud.google.com/cpp/reference/connectors/latest",
+    "distribution_name": "google-cloud-cpp",
+    "issue_tracker": "https://issuetracker.google.com/savedsearches/5929031",
+    "language": "cpp",
+    "library_type": "GAPIC_AUTO",
+    "name_pretty": "Connectors API",
```

Comment:
maybe: "Apigee Connectors API"

The branding list does not endorse "Connectors API" or "Apigee Connectors API", FWIW. The closest thing that shows up in the list is "Apigee Integration", but eh.

Reply:
I think this is too low level for anybody to care about branding.  In any case:

https://github.com/googleapis/googleapis/blob/ef2e2ea532248d6dc40a56bc6c95cea858ba31b6/google/cloud/connectors/v1/connectors_v1.yaml#L4